### PR TITLE
[ANCHOR-292] Separate the platform API endpoints from the sep server

### PR DESCRIPTION
**Pull request by lijamie98**
<details>
  <summary>PR Checklist</summary>

### PR Structure

* [ ] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [ ] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [ ] This PR's title starts with name of package that is most changed in the PR, ex.
  `paymentservice.stellar`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
</details>

### What

Separate the platform API endpoints from the sep server.

### Why

The platform API endpoints are sensitive and should not be exposed with the SEP endpoints which are public.

### CODE REVIEW

File: .run/Run - Platform Server - no Docker.run.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Run - Platform Server - no Docker" type="JetRunConfigurationType">
```
Comment (paulormnas) [inline .run/Run - Platform Server - no Docker.run.xml:2]: @lijamie98, as you updated the other run configurations from `Docker` to `fullstack`, shouldn't you do the same here?

File: .run/Run - Sep Server - no Docker.run.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <component name="ProjectRunConfigurationManager">
-  <configuration default="false" name="Run - SepServer - no Docker" type="JetRunConfigurationType">
+  <configuration default="false" name="Run - Sep Server - no Docker" type="JetRunConfigurationType">
```
Comment (paulormnas) [inline .run/Run - Sep Server - no Docker.run.xml:2]: Same thing here.

**Comment by philipliu** [comment]
There are some references to port 8080 in some configs/examples. Do we need to update those as well? 

For example, https://github.com/stellar/java-stellar-anchor-sdk/blob/496d8c9c39052dfdb3ad4a8736bd42ef2a18fdc3/platform/src/main/resources/example.anchor-config.yaml#L54-L58

**Comment by lijamie98** [comment]
> There are some references to port 8080 in some configs/examples. Do we need to update those as well?
> 
> For example,
> 
> https://github.com/stellar/java-stellar-anchor-sdk/blob/496d8c9c39052dfdb3ad4a8736bd42ef2a18fdc3/platform/src/main/resources/example.anchor-config.yaml#L54-L58

These ports: `8080` are correct because they are pointing to the `sep` server. Port `8085` is used internally for `platform`.

File: docs/00 - Stellar Anchor Platform.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [![License](https://badgen.net/badge/license/Apache%202/blue?icon=github&label=License)](https://github.com/stellar/java-stellar-anchor-sdk/blob/develop/LICENSE)
 [![GitHub Version](https://badgen.net/github/release/stellar/java-stellar-anchor-sdk?icon=github&label=Latest%20release)](https://github.com/stellar/java-stellar-anchor-sdk/releases)
-[![Docker](https://badgen.net/badge/Latest%20Release/v1.1.0/blue?icon=docker)](https://hub.docker.com/layers/stellar/anchor-platform/1.1.0/images/sha256-c9df239014d31a29af4e379aaf035e489cabb4c51623d135500022a10d136fbd?context=explore)
+[![Docker](https://badgen.net/badge/Latest%20Release/v2.0.1/blue?icon=docker)](https://hub.docker.com/layers/stellar/anchor-platform/2.0.1/images/sha256-bce3ee60276cef20cb428cf50eed829d9833a9149fa2aedf63c2373f7a40e8a1)
```
[the remark this answers is not in the document]
Comment (lijamie98) [reply docs/00 - Stellar Anchor Platform.md]: Changed the docker version back to `1.2.13`.

File: integration-tests/src/test/kotlin/org/stellar/anchor/platform/test/PlatformApiTests.kt [hunk 4]
```diff
@@ -11,18 +9,7 @@ class PlatformApiTests(config: TestConfig, toml: TomlContent, jwt: String) {
   private val platformApiClient =
     PlatformApiClient(AuthHelper.forNone(), config.env["platform.server.url"]!!)
 
-  private fun testHealth() {
```
[the remark this answers is not in the document]
Comment (lijamie98) [reply integration-tests/src/test/kotlin/org/stellar/anchor/platform/test/PlatformApiTests.kt:14]: And we may potentially move all the health tests to a separate test class.

File: platform/src/main/java/org/stellar/anchor/platform/AbstractPlatformServer.java [hunk 5]
```diff
@@ -21,4 +23,11 @@ public void stop() {
       ctx = null;
     }
   }
+
+  public void addFormatters(FormatterRegistry registry) {
```
Comment (Ifropc) [inline platform/src/main/java/org/stellar/anchor/platform/AbstractPlatformServer.java:27]: FYI: this formatters are only for platform server

File: integration-tests/src/test/kotlin/org/stellar/anchor/platform/test/SepHealthTests.kt [hunk 6]
```diff
@@ -0,0 +1,28 @@
+package org.stellar.anchor.platform.test
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertNotNull
+import org.stellar.anchor.apiclient.PlatformApiClient
+import org.stellar.anchor.auth.AuthHelper
+import org.stellar.anchor.platform.TestConfig
+import org.stellar.anchor.util.Sep1Helper.TomlContent
+
+class SepHealthTests(config: TestConfig, toml: TomlContent, jwt: String) {
```
Comment (Ifropc) [inline integration-tests/src/test/kotlin/org/stellar/anchor/platform/test/SepHealthTests.kt:10]: Do we need a health endpoint to be added to platform server too?

**Pull request merged**